docs/content/docs/patterns/observability/_index.md

```diff
@@ -10,6 +10,6 @@ aliases: ["observability"]
 OSM's observability stack includes Prometheus for metrics collection, Grafana for metrics visualization, Jaeger for tracing and Fluent Bit for log forwarding to a user-defined endpoint. These are disabled by default but may be enabled during OSM installation with flags `--deploy-prometheus`, `--deploy-grafana`, `--deploy-jaeger` and `--enable-fluentbit`.
 
 ## Table of Contents
-- [Metrics - Prometheus and Grafana](./metrics.md)
+- [Metrics - Prometheus and Grafana](/docs/tasks_usage/metrics.md)
 - [Tracing - Jaeger](./tracing.md)
-- [Log forwarding - Fluent Bit](./logs.md)
\ No newline at end of file
+- [Log forwarding - Fluent Bit](/docs/tasks_usage/logs.md)
```

Comment:
Per the current readme (https://github.com/openservicemesh/osm/tree/main/docs#external-links), anything outside `/docs/content` should use absolute URLs. Is this still the case or does the readme need to be updated?

Reply:
That's what I'm experimenting to understand. I don't seem to have it entirely right, yet (because of the differences between https://deploy-preview-2636--osm-docs.netlify.app/docs/tasks_usage/metrics.md (not working on website, only github) and https://deploy-preview-2636--osm-docs.netlify.app/docs/tasks_usage/metrics/).

Reply:
I approved it so you can merge and keep testing :-)

Reply:
Related issue that is open: https://github.com/openservicemesh/osm/issues/2453

Reply:
Ah. It's not intuitively obvious at all, but does work as originally documented.

Reply:
Oh! I just realized what you meant - this is _not_ `anything outside /docs/content` - because we (somewhat confusingly) have a path that looks like this: https://github.com/openservicemesh/osm/tree/main/docs/content/docs/tasks_usage

so the root where you can start saying `/` is _under_ https://github.com/openservicemesh/osm/tree/main/docs/content/ and you'll see the word "docs" there _again_!